recipe/meta.yaml

```diff
@@ -10,21 +10,22 @@ source:
   md5: 9c2ad6cf86fe4a7bc0de8d2d92b101f4
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win]
 
 requirements:
   build:
-    - fftw
-    - gcc
+    - {{ compiler('c') }}
+    - openmp  # [osx]
```

Comment:
Thinks this needs to be in `host`. Also should be in `run` as well.

xref: https://github.com/conda-forge/python-spams-feedstock/commit/b9801062aa56c9220f6577e32995bc4a4e660b28

Reply:
Thanks for the heads-up and the reference!